pyccel/codegen/printing/fcode.py

```diff
@@ -1225,6 +1225,15 @@ def _print_String(self, expr):
     def _print_Interface(self, expr):
         # ... we don't print 'hidden' functions
         name = self._print(expr.name)
+        if expr.is_argument:
+            sigs = ''
+            for f in expr.functions:
+                self._handle_fortran_specific_a_prioris(list(f.arguments) + list(f.results))
+                sigs += self.function_signature(f, f.name) + '\n'
+                sigs += 'end %s\n' % sigs.split()[0]
```

Comment:
It would be nice to separate the functions' signatures with an empty line, and to properly indent them.

@EmilyBourne Should we add the `interface` keyword to the `indent_code` function?

Reply:
Yes I think so